### ISSUE TITLE
Regarding Eq11 in paper

### ISSUE THREAD

**Issue by yitongx** [ChaoyiZh/latent-radiance-field#8]
Hi, 
First of all, thanks for sharing your paper and codes.  

I have questions about your Eq 11 in paper, regarding loss term for the novel view decoded 2D latent.

I haven't dive into the codes so I'm gonna ask directly: 

How do you get the GT image for novel views? Did you use the testing views in the dataset for training? And then evaluate your metrics on the same views when testing? 

Thanks in advance.

**Comment by ChaoyiZh** [comment]
Hi,
Thanks a lot for your interest in our work!

Regarding your question on the dataset and the GT images for novel views, you can refer to **Appendix A.2 "DETAILS OF DATASET"** in the paper for more specifics.

To clarify:
We randomly selected a subset of scenes from the DL3DV-10K dataset as our **training set**. During Stage-III (decoder fine-tuning), we used both the train and test views within these training scenes to train the decoder. This strategy helps improve the robustness of our decoder across different viewpoints.

For the **Latent NVS experiments**, we sampled test scenes from DL3DV-10K that **do not overlap** with the training scenes at all. The evaluation metrics are computed on these unseen test scenes to ensure a fair evaluation.

Hope this clears things up! Feel free to ask if you have more questions.

**Comment by yitongx** [comment]
Oh okay. I will check the appendix and dataset. 

Thanks for your explanation.
